cartesi-rollups/development/running-the-application.md

```diff
@@ -51,12 +51,18 @@ TX_DEFAULT_CONFIRMATIONS=1
 
 ## NoNodo (Testing & Development)
 
-NoNodo is a development node for Cartesi Rollups designed to work with applications running in the host machine instead of the Cartesi machine. With NoNodo, you don’t need to run Docker or compile the application to RISC-V.
+NoNodo is a development tool for Cartesi Rollups designed to work with applications running in the host machine instead of the Cartesi machine. With NoNodo, you don’t need to run Docker or compile the application to RISC-V.
 
-:::note Prerequisites
-Set up Anvil on your system. Refer to [the instructions in the Foundry book for Anvil installation](https://book.getfoundry.sh/anvil/) details.
+:::note
+Applications may require RISC-V compilation for interpreted languages. NoNodo allows unrestricted operations outside Cartesi's sandbox but lacks support for Cartesi-specific behaviors like full machine reversion.
+
+Additionally, NoNodo exclusively supports applications utilizing the Cartesi Rollups HTTP API, with performance degradation and input delays compared to running locally.
```

Comment:
I don't understand this whole sentence. Running NoNodo is the most performant will ever get (under current tech).

Reply:
I have made a commit to change this to:

`Applications may require RISC-V compilation for interpreted languages. NoNodo works for applications that use the Cartesi Rollups HTTP API and doesn't work with applications using the low-level API. 
NoNodo offers an improved speed and performance compared to running your application inside a Cartesi machine.
`

Reply:
maybe, you want to keep it short:

"Applications may still require RISC-V compilation even interpreted languages, as they use libraries with binaries for x64 platforms.